ci-operator/step-registry/ipi/conf/ibmcloud/kms-key/ipi-conf-ibmcloud-kms-key-commands.sh

```diff
@@ -6,10 +6,79 @@ set -o pipefail
 
 CONFIG="${SHARED_DIR}/install-config.yaml"
 
-cat "${SHARED_DIR}/ibm_kpKey.yaml"
+echo "ControlPlane EncryptionKey: ${IBMCLOUD_CONTROL_PLANE_ENCRYPTION_KEY}"
+echo "Compute EncryptionKey: ${IBMCLOUD_COMPUTE_ENCRYPTION_KEY}"
+echo "DefaultMachinePlatform EncryptionKey: ${IBMCLOUD_DEFAULT_MACHINE_ENCRYPTION_KEY}"
 
-yq-go m -x -i "${CONFIG}" "${SHARED_DIR}/ibm_kpKey.yaml"
+key_file="${SHARED_DIR}/ibmcloud_key.json"
+if [[ ! -f "${key_file}" ]]; then
```

Comment:
I think no need the if condition, user should be responsible for calling `ibmcloud-provision-kms-key` to create this json file.